images/scripts/rhel.setup

```diff
@@ -306,6 +306,12 @@ else
     dnf --setopt=install_weak_deps=False install -y cockpit
 fi
 
+# HACK: 8.4 has broken NTP
+# https://bugzilla.redhat.com/show_bug.cgi?id=1918340
+if [ "$IMAGE" = "rhel-8-4" ]; then
+    dnf install -y timedatex
+fi
```

Comment:
Hmm, I'd just have added it to `TEST_PACKAGES`. It's not a hack around an OS deficiency, it's just an assumption of cockpit's tests IMHO. (fine for me to not rebuild the image for that, as the end result is the same)

Reply:
Oh, nevermind, I should have read the bug first. Sorry!